db/migrate/20231129150911_create_likes.rb

```diff
@@ -0,0 +1,11 @@
+class CreateLikes < ActiveRecord::Migration[7.1]
+  def change
+    create_table :likes do |t|
+      t.references :user, null: false, foreign_key: true
+      t.references :post, null: false, foreign_key: true
+      t.string :text
+
+      t.timestamps
+    end
+  end
```

Comment:
- [ ] As required for the project and according to the the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd_v1_1.png) given to you, kindly ensure that your `likes` table has the correct columns as the `text` column has to be removed from the table.